pkg/server/sandbox_stop.go

```diff
@@ -105,10 +106,10 @@ func (c *criService) stopSandboxContainer(ctx context.Context, sandbox sandboxst
 		return errors.Wrap(err, "failed to get sandbox container")
 	}
 
-	// Delete the sandbox container from containerd.
-	_, err = task.Delete(ctx, containerd.WithProcessKill)
+	// Kill the sanbox container.
```

Comment:
/s/san/sand/

Reply:
Please update function comment on line 98 to:
// stopSandboxContainer kills the sandbox container.
// note: task.Delete is not called here because it will be called when the event monitor handles the TaskExit event

Reply:
Will do.

Reply:
Done.